stdpopsim/models.py

```diff
@@ -259,3 +261,41 @@ def __init__(self, N0, *args):
         for t, N in args:
             self.demographic_events.append(msprime.PopulationParametersChange(
                 time=t, initial_size=N, growth_rate=0, population_id=0))
+
+
+class GenericIM(Model):
+    """
+    Takes parameters N0, N1, N2, T, M12, and M21, as list of arguments:
+    (N0, N1, N2, T, M12, M21).
+    """
+    id = "IM"
+    name = "Isolation with migration"
+    description = """
+        A generic isolation with migration model where a single population of
+        size N0 splits into two populations of constant size N1 and N2 time T
+        generations ago, with migration rates M12 and M21 between the split
+        populations.
+        """
+    citations = []
+    populations = [_pop0, _pop1, _pop2]
+    author = None
+    year = None
+    doi = None
+
+    def __init__(self, *args):
```

Comment:
Might as well put these into the arguments explicity, rather than unpacking (I don't think it's right atthe moment anyway)

Reply:
Right - hurrying through and forgot the tests.

And yeah, arguments could be explicit. Will get to it shortly

Reply:
yeah agree this is the correct place